### DISCUSSION TITLE
Tesla custom component - Configuring unit of measurement

### DISCUSSION THREAD

**Discussion by nil-ghosh** [alandtse/tesla#498] [Ideas]
Could it be possible to configure the unit of measurement (km or miles)  as per user requirement?

**Comment by nil-ghosh** [comment]
> Could it be possible to configure the unit of measurement (km or miles) as per user requirement?

This could be closed. Figured out a way to do it.

**Comment by mgrefsrud** [comment]
Could you please share how you did it? I would also like to change this.

**Reply by nil-ghosh** [reply]
If you select the entity in Device and Entities, then go to Settings, there’s a field to select your Unit of Measurement. 

Hope that helps

cheers

**Reply by mgrefsrud** [reply]
Yes there it was, Thank you!
